### PR TITLE
Home page strapline component

### LINKED ISSUE #1829: The homepage contains a very concise description of what PlaceCal is (tag line)
## Acceptance criteria

- [ ] There is a feature area on the homepage with concise text (use text from site for now)
> PlaceCal is a community directory and events calendar where you can find out everything that’s happening in your neighbourhood all in one place.
- [ ] Replace temp copy in "concise description" box on the homepage
- [ ] The component is implemented as a view component
- [ ] The design matches the image below (without button?)

## implementation plan

- no white title as it's extra words and doesn't meet acceptable contrast threshold
- no button as we have a link in the nav
- use this design
- checkout commit 1abc4fe7 if you would like to see how the original version was implemented

Screenshot 2023-05-04 at 14-54-43 The homepage contains a very concise description of what PlaceCal is (tag line) · Issue #1829 · geeksforsocialchange_PlaceCal

### PR DESCRIPTION
**Pull request by ivan-kocienski-gfsc**
Closes #1829 

- has text from issue
- uses view_components

Is this how view components should be built?

### CODE REVIEW

**Comment by katjam** [comment]
@ivankocienski Posting screenshots here

**Comment by ivan-kocienski-gfsc** [comment]
# Desktop

Screenshot_20230505_144755

# Mobile Portrait

Screenshot_20230509_122031

# Mobile Landscape

Screenshot_20230509_122311

**Comment by katjam** [comment]
Do you have any without the button?

**Comment by kimadactyl** [comment]
looking good but needs a little bit of space around the outside i think - check live site for ref!

**Comment by katjam** [comment]
Also noting feedback from David - I think if we don't know where the button goes, let's drop it. We can always put it back when the copy gets finalised.

image

**Comment by ivan-kocienski-gfsc** [comment]
I have removed the CTA button and the BG images

**Comment by studiosquid** [comment]
Have attached some reference images to show how the header adapted in the 2020 design (basically it was super bespoke around the text size) but it also shows the spacing around the illustration at the various different points :) 

I'm also quite uncertain the illustrations are on message for 2023. 

2020 Edition 320 – Home

2020 Edition Narrow tablet – Home

2020 Edition iPad Portrait – Home

2020 Edition iPad Landscape – Home


2020 Edition Web 1366 – Home with pics

**Comment by ivan-kocienski-gfsc** [comment]
I am going to fix the padding and spacing.

Do we want the images back?

Is the text copy good to go?

Would it be possible to merge this as a first pass and open an issue that details the specific changes required w/r/t graphic/layout ?

File: app/components/home_strapline_component/home_strapline_component.sass [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  
+  &-quote
+    font-size: $h3-size
+    margin: 0 auto
+    text-align: center
+    padding: 1rem 1rem
+
+    +for-tablet-portrait-up
+      width: 50%
+      line-height: 1.6em
```
Comment (aaaaargZombies) [inline app/components/home_strapline_component/home_strapline_component.sass:18]:
```suggestion
      width: 50%
      line-height: 1.6em
      font-size: 1.66rem
```

**Review by aaaaargZombies — APPROVED** [review]
hey I'm approving in the interest of getting this merged, I think your right that adding more things like images etc should be future issues.

I have one request that the font size is increased on large devices based on some similiar fonts in the XD file, I've used the suggestion thing so you *should* just be able to click the button.

**Pull request merged**